src/get.rs

```diff
@@ -60,6 +60,17 @@ pub extern "C" fn z_reply_err_payload(this_: &z_loaned_reply_err_t) -> &z_loaned
     this_.as_rust_type_ref().payload().as_loaned_c_type_ref()
 }
 
+/// Returns mutable reply error payload.
+#[no_mangle]
+pub extern "C" fn z_reply_err_payload_mut(
```

Comment:
There is no way to get mutable err or mutable sample from the reply, so this method is useless for now.

Reply:
Then we should add this. SHM buffer may reside literally everywhere where ZBytes is

Reply:
added